### PR TITLE
Tabs should only scroll on a horizontal TabBarView fling

### PR DESCRIPTION
**Pull request by HansMuller**

### CODE REVIEW

File: packages/flutter/lib/src/material/tabs.dart [hunk 1]
```diff
@@ -31,6 +31,10 @@ const EdgeDims _kTabLabelPadding = const EdgeDims.symmetric(horizontal: 12.0);
 const double _kTabBarScrollDrag = 0.025;
 const Duration _kTabBarScroll = const Duration(milliseconds: 300);
 
+// The scrollOffset (velocity) provided to fling() is pixels/ms, and the
+// tolerance velocity is pixels/sec.
```
Comment (Hixie) [inline packages/flutter/lib/src/material/tabs.dart]: this comment is great. it doesn't explain the "2" though...

**Comment by Hixie** [comment]
lgtm with or without the comment change

File: packages/flutter/lib/src/material/tabs.dart (hunk 1 above)
Comment (HansMuller) [inline packages/flutter/lib/src/material/tabs.dart]: I'm afraid that the factor of 2 was determined experimentally.  I didn't know what to say...

Comment (Hixie) [inline packages/flutter/lib/src/material/tabs.dart]: I think saying that is fine.

**Pull request merged**